include/s3select.h

```diff
@@ -3190,18 +3190,25 @@ class json_object : public base_s3object
 	status = -1;
       }
 
+      ++m_row_count;
+
       if(is_sql_limit_reached()) 
       {
 	      status = JSON_PROCESSING_LIMIT_REACHED;//returning number since sql_execution_on_row_cb is a callback; the caller can not access the object
+        m_sql_processing_status = s3selectEngine::base_s3object::Status::LIMIT_REACHED;
```

Comment:
reviewing function 
`is_sql_limit_reached` 
https://github.com/ceph/s3select/blob/4eb782c028d210f3af7407467ec878d2a84457d4/include/s3select.h#L2223C8-L2223C28

this line is obsolete.